core/metacling/test/TClingCallFuncTests.cxx

```diff
@@ -14,6 +16,19 @@
 // system, feel free to delete them as these tests here don't represent things the user
 // should do in his code.
 
+class FilterDiagsRAII {
```

Comment:
Thanks. That seems quite useful. Can you add it in this header https://github.com/root-project/root/blob/master/test/unit_testing_support/ROOTUnitTestSupport.h